plugin/src/main/java/org/websync/WebSyncService.java

```diff
@@ -96,4 +96,20 @@ public void removeProject(Project project) {
         this.provider.removeProject(project);
     }
 
+    public void updateComponentInstance(String className, String oldFieldName, String newFieldName) {
```

Comment:
not sure that this class has this responsibility

Reply:
In general, I agree with you. But now it's the best place for it. The next step is redesign the org.websync.websession.psimodels package and provide the correct way to store and update PSI structures in the IDE.